app/models/experiment.rb

```diff
@@ -335,10 +336,10 @@ def deep_clone_to_project(current_user, project)
     format = 'Clone %d - %s'
 
     i = 1
-    i += 1 while experiment_names.include?(format(format, i, name)[0, 50])
+    i += 1 while experiment_names.include?(format(format, i, name))
 
     clone = Experiment.new(
-      name: format(format, i, name)[0, 50],
```

Comment:
What is that [0, 50] needed? It is OK to get rid of it? Do we have to truncate the name here?